### PR TITLE
Fixed the issue #56

### PR DESCRIPTION
**Pull request by CalDev98**
Just removed the Radius at the bottom of the drawer

### CODE REVIEW

**Comment by dadiorchen** [comment]
@CalDev98 can you resolve the conflict first than push your code again, (no need to create new PR)
Usually, I will `git merge .../master` and fix the conflict on my side than push code again.

File: .env [hunk 1]
```diff
@@ -1,3 +1,4 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (dadiorchen) [inline .env:1]: @CalDev98 what's this setting for? Why should we add it

Comment (CalDev98) [reply .env:1]: > @CalDev98 what's this setting for? Why should we add it

I just used it because of my webpack version

**Comment by CalDev98** [comment]
> @CalDev98 can you resolve the conflict first than push your code again, (no need to create new PR)
> Usually, I will `git merge .../master` and fix the conflict on my side than push code again.
> 
> image

@dadiorchen I think everything is solved now.

**Comment by dadiorchen** [comment]
@CalDev98 good job!

**Pull request merged**